doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -701,6 +701,7 @@ Libraries for NFC
 -----------------
 
 * Added an experimental serialization of NFC tag 2 and tag 4 APIs.
+* Fixed a potential issue with handling data pointer in the function :c:func:`ring_buf_get_data` in the :file:`platform_internal_thread` file.
```

Comment:
```suggestion
* Fixed a potential issue with handling data pointers in the function :c:func:`ring_buf_get_data` in the :file:`platform_internal_thread` file.
```
Maybe data pointers? Also the "ring_buf_get_data" function doesn't link to the API documentation. Could you please check this.

Reply:
The function is static, so internal. It shouldn't be documented. Should I change the link or maybe omit the name entirely?

Reply:
We could represent the function in the double backticks. I have updated my comments. Please check.